src/hangar/cli/utils.py

```diff
@@ -0,0 +1,10 @@
+
+def parse_custom_arguments(click_args):
```

Comment:
What does this function do? I'm confused. Why does it need it's own module?

Reply:
Keeping `utils` there for any future utils that we could have (in fact there might be existing logic in cli.py which could be moved to utils). 
`parse_custom_arguments` is the function that takes the unknown arguments from the command line (https://github.com/hhsecond/hangar-py/blob/9d70a533f92a3c0efba1cec7d0b49ff345b3cd16/src/hangar/cli/cli.py#L364) which is plugin-specific. For example, if the user wants to pass the dimension of images to be while doing the import, he could pass `--height` and `--width` if plugin is designed to work on those. Does this make sense?

Reply:
yup! good idea for this one. Thanks!